### ISSUE TITLE
Framework Search Paths in compile_commands.json is not used

### ISSUE THREAD

**Issue by heshuimu** [microsoft/vscode-cpptools#2508]
**Type: LanguageService**

**Describe the bug**
- OS and Version: macOS 10.13.6
- VS Code Version: VSCode 1.27.1
- C/C++ Extension Version: 0.18.1
- Other extensions you installed (and if the issue persists after disabling them): CMake, CMake Tools
- A clear and concise description of what the bug is: 

When using `compile_commands.json` generated by CMake and a custom Mac framework is included, framework search path in the form of `-F/path/to/framework/parent/directory` will present in the build command. But C++ Plugin does not include this during IntelliSense which gives squiggles wherever the framework header is used. 

**To Reproduce**

1. Create a CMake project with a C++ source file and include a custom iOS framework with `include_directories`. 
2. In the source file, include the framework header with the ordinary syntax. 
2. In VSCode, use plugin "CMake Tools" to build the CMake project. 
3. Set `c_cpp_properties.json` for project to use `compile_commands.json` generated by CMake
4. (Restart VSCode to refresh, ) Open the source file and look for squiggle that says "source file is not found"

**Expected behavior**

The framework header should be recognized and IntelliSense should work as expected. 

**Additional context**
```
...
  include: /usr/include
  framework: /System/Library/Frameworks
  framework: /Library/Frameworks
<THERE SHOULD HAVE BEEN framework: /path/to/framework/parent/directory>
  define: __llvm__=1
...
```

**Comment by heshuimu** [comment]
**Workaround:** 

Add the framework to `/Library/Frameworks` by copying or symlinking.

**Comment by bobbrow** [comment]
I added support for `-F` and `-framework`.  Will be available in 0.19.1 released later this week.
